rules/sonic-yang-mgmt-py2.mk

```diff
@@ -0,0 +1,8 @@
+# sonic-yang-mgmt python2 wheel
+
+SONIC_YANG_MGMT_PY2 = sonic_yang_mgmt-1.0-py2-none-any.whl
```

Comment:
Let us not tag it PY2.  We have to move to Python3 in 201911 image. So at least call .py and when system default switches to python3, we will use it transparently.